packages/patternfly-4/gatsby-theme-patternfly-org/components/propsTable/propsTable.js

```diff
@@ -34,7 +34,9 @@ export const PropsTable = props => {
       caption={props.caption}
       rows={props.rows.map(row => ({
         cells: [
-          row.name,
+          <div className='pf-m-fit-content'>
```

Comment:
Funny enough, I think we had done this before at some point. Don't remember why we reverted it.

Reply:
Also, I'm surprised that single quotes in `className='pf-m-fit-content'` works! I'm used to HTML requiring double quotes. I think this is a "feature" of JSX, but until we lint this code I don't mind :)

Reply:
Yeah I originally . had double quotes, but then saw the rest of the file was using single so I followed the pattern...